Golint error:
```
staging/src/k8s.io/component-base/logs/options.go:61:36: error strings should not be capitalized or end with punctuation or a newline
```

--- a/staging/src/k8s.io/component-base/logs/options.go
+++ b/staging/src/k8s.io/component-base/logs/options.go
@@ -58,7 +58,7 @@ func (o *Options) Validate() []error {
 		allFlags := unsupportedLoggingFlags()
 		for _, fname := range allFlags {
 			if flagIsSet(fname) {
-				errs = append(errs, fmt.Errorf("Non-default logging format doesn't honor flag: %s\n", fname))
+				errs = append(errs, fmt.Errorf("non-default logging format doesn't honor flag: %s", fname))
 			}
 		}
 	}

